source/rates_rest_api.html.md

```diff
@@ -0,0 +1,142 @@
+# Introduction
+
+The Clear Street rates service enables users to request the borrow rate for a security
+
+# API Endpoints
+
+## Rates
+
+### HTTP Request
+
+`POST - /rates`
+
+```
+POST - /rates
+```
+
+1. Client requests the current borrow rate for a security
+
+2. Clear Street responds with the borrow rate for the matching criteria
+
+<aside class="notice">
+Borrow rates can only be fetched during business hours starting from 4:00 AM to 10:00 PM EST.
+To prevent excessive use, the limit is approximately 10,000 requests per day. Please contact us if you wish to request an increase.
+</aside>
+
+### Request Object
+
+| Parameter     | Type   | Description                                                                           |
+| ------------- | ------ | ------------------------------------------------------------------------------------- |
+| mpid          | string | **REQUIRED**. The Market Participant Identifier, assigned by Clear Street (e.g. CLST) |
+| security_type | string | **REQUIRED**. The type of security (e.g. Ticker, FIGI, CUSIP, ISIN, or SEDOL)         |
+| security      | string | **REQUIRED**. The security identifier                                                 |
+
+> Request Object
+
+```json
+{
+  "mpid": "AAAA",
+  "security_type": "TICKER",
+  "security": "SPY"
+}
+```
+
+### Responses
+
+<details>
+  <summary><aside class="success">200 OK</aside></summary>
+
+A successful response
+
+```json
+{
+  "rate_id": "d18adab0-1527-11ee-be56-0242ac120002",
+  "mpid": "AAAA",
+  "security_type": "TICKER",
+  "security": "SPY",
+  "borrowRate": 5.183
```

Comment:
pls respond with borrow rates in bps as a whole number

Reply:
So I believe Monte wants to go into a further discussion about this. I've been told that Michael Batanjany would like to keep the decimal but change the precision to 2 significant digits.